fly.toml.sample

```diff
@@ -0,0 +1,46 @@
+kill_signal = "SIGTERM"
+kill_timeout = 5
+processes = []
+
+[env]
+  INTEGRATED_TCP_TURN_PORT = "49999"
+  INTEGRATED_TURN_IP = "127.0.0.1"
```

Comment:
We are missing INTEGRATED_TURN_PROD_IP, and instead of  `127.0.0.1`  let's write `<ip obtained with fly ips allocate-v4>`